x-pack/plugins/lists/public/exceptions/hooks/use_api.ts

```diff
@@ -105,6 +110,71 @@ export const useApi = (http: HttpStart): ExceptionsApi => {
           onError(error);
         }
       },
+      async getExceptionListsItems({
+        lists,
+        filterOptions,
+        pagination,
+        showDetectionsListsOnly,
+        showEndpointListsOnly,
+        onSuccess,
+        onError,
+      }: ApiCallFindListsItemsMemoProps): Promise<void> {
+        const abortCtrl = new AbortController();
+        const { ids, namespaces } = lists
+          .filter((list) => {
+            if (showDetectionsListsOnly) {
```

Comment:
This logic of filter and reduce looks like is duplicated elsewhere. Just mentioning in case you want to make it into a utility, but am fine if it needs to be duplicated here and later on but would be prone to possible changes/bug fixes in one vs. the other later.

Reply:
Moved to util and added test!